Automation/SQL/azureDeploy.parameters.json

```diff
@@ -0,0 +1,16 @@
+
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "serverName": {
+          "value": "GEN-UNIQUE"
+      },
+      "administratorLogin": {
+          "value": "GEN-UNIQUE"
```

Comment:
same here - use "defaultValue" if anything

Reply:
I just deleted that file :)